packages/client/tests/functional/_utils/setupTestSuiteEnv.ts

```diff
@@ -106,12 +107,30 @@ export async function setupTestSuiteDatabase(
   suiteMeta: TestSuiteMeta,
   suiteConfig: NamedTestSuiteConfig,
   errors: Error[] = [],
+  alterStatement?: string,
 ) {
   const schemaPath = getTestSuiteSchemaPath(suiteMeta, suiteConfig)
 
   try {
     const consoleInfoMock = jest.spyOn(console, 'info').mockImplementation()
     await DbPush.new().parse(['--schema', schemaPath, '--force-reset', '--skip-generate'])
+
+    if (alterStatement) {
+      const prismaDir = path.dirname(schemaPath)
+      const timestamp = new Date().getTime()
+
+      await fs.promises.mkdir(`${prismaDir}/migrations/${timestamp}`, { recursive: true })
+      await fs.promises.writeFile(`${prismaDir}/migrations/migration_lock.toml`, 'provider = "postgresql"')
```

Comment:
We should use the provider dynamically here, I think? Is this file really needed?

Reply:
Sure!